test/some.jl

```diff
@@ -49,6 +49,34 @@
 @test !isequal(Some(1), nothing)
 @test !isequal(Some(nothing), nothing)
 
+# Some with something else is false
+@test !=(Some(nothing), nothing)
+@test !=(nothing, Some(nothing))
+
+# Two Somes forward to their wrapped things
+@test ==(Some([0x1]), Some([1]))
+
+# Don't propagate wrapped missings
```

Comment:
```suggestion
# Don't propagate wrapped `missing`s
```
Same as above, though no idea if this will actually fool it

Reply:
Seems to have worked!